app/components/markdown/markdown.tsx

```diff
@@ -403,16 +405,24 @@ const Markdown = ({
         );
     };
 
-    const renderLink = ({children, href}: {children: ReactElement; href: string}) => {
+    const renderLink = ({children, href}: {children: any; href: string}) => {
         if (isUnsafeLinksPost) {
             return renderText({context: [], literal: href});
         }
+
+        const childrenWithLinkFlag = React.Children.map(children, (child) => {
+            if (React.isValidElement(child)) {
+                return React.cloneElement(child as ReactElement<any>, {isInsideLink: true});
+            }
+            return child;
+        });
+
```

Comment:
You actually shouldn't need to add `isInsideLink` all since the existing `context` prop can be used for that. Take a look at how `renderImage` detects that the image is inside of a table since I think the same thing should work if it's inside of a link

Reply:
Great catch guys! That makes total sense, I tried to see if there were code similar to it so I don't a prop, but didn't saw the `indexOf` condition.

It works great! And now we don't have to add the complexity of adding a new prop to the entire tree.